### ISSUE TITLE
PHP8: Inconsistent formatting when using attributes

### ISSUE THREAD

**Issue by markkirchner** [prettier/plugin-php#1870]
**Prettier 2.4.1**

**PHP Plugin 0.17.6**

**Input:**

```php
use JetBrains\PhpStorm\Pure;

class Test
{
    /**
     * Method with an attribute.
     * @param string $foo
     * @return string
     */
    #[Pure]
    public function withAttribute(string $foo): string
    {
        return $foo;
    }

    /**
     * Method without any attributes.
     * @param string $foo
     * @return string
     */
    public function withoutAttribute(string $foo): string
    {
        return $foo;
    }
}
```

**Expected:**
Code is already correctly formatted => runnning Prettier should not result in any changes.

**Result:**
The opening brace of the method that contains the attribute gets moved, the other one does not.
```php
    public function withAttribute(string $foo): string {
        return $foo;
    }
```

**Additional note:**
Removing the `string $foo` parameter from the method signature makes the problem go away. So it seems to be a rather weird combination of factors that trigger the bug.

**Comment by tm1000** [comment]
Same thing is happening here too.

Here's the playground for reference: https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCDoA6AIwgWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDaRTtrMMEEJvBAAHgB+YzAAHUwt4SgAAgApWAAhVzqNjYCTRP6fC4DBaABuLa2Y2THsPdEoca3gLZ7IF7AD0ACowYDgXswXsALKwajoCh7ADuvWoezYmCxcAg8A48ygtihwNhAAEwGxXD49uN8ZhlHsACRSdDoUlAinQGCPHH0lScmEgzkAYgA2g9oABdTlzDgeGJ7KSCTAxDA49E0ACCeIJRIAFALGSy2egAJR4OkuQU44EAu3QoE8vmm9kvR17dqvT3gyGemHwxHItEY9DzLGYJK4m2EnTYEkBilUmnWhlM1nsoXc2Cu43KbMiz3yxXK1XqoaasPzXWxw35t0Wq35zkOp3Al0QHGZ9Ae6HezDtEDtIA

**Comment by tm1000** [comment]
@czosel what if we filter out attributes from `declaration` before sending it to `some(willBreak)`

**Comment by oliverschwendener** [comment]
I am experiencing the same issue. Would be cool to see an update regarding this.

**Comment by czosel** [comment]
@tm1000 Sounds good - PRs welcome! :)

**Comment by cseufert** [comment]
I think I have figured this out, we including the attrs as part of the is this a multiline function declaration check.

**Comment by czosel** [comment]
Fixed in `v0.19.2` :tada:
